dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/BeamSQLFnExecutor.java

```diff
@@ -74,7 +79,7 @@ public BeamSQLFnExecutor(BeamRelNode relNode) {
    * {@link #buildExpression(RexNode)} visits the operands of {@link RexNode} recursively,
    * and represent each {@link SqlOperator} with a corresponding {@link BeamSqlExpression}.
    */
-  private BeamSqlExpression buildExpression(RexNode rexNode) {
+  static BeamSqlExpression buildExpression(RexNode rexNode) {
```

Comment:
Remove `private` make it `package` level access. Useful for testing, testing class within the same package can see this method, but classes in other packages can not see this method, right?